### PR TITLE
[TP-1856/1857] Feat: add DEX SDK mainnet configuration

### PR DESCRIPTION
**Pull request by pano-skylakis**
# Summary

Adds Immutable zkEVM mainnet configuration for the DEX SDK

## Security

# Things worth calling out

# Before submitting the PR, please consider the following:

- [ ] Prefix your PR title with `feat: `, `fix: `, `chore: `, `docs:`, or `refactor:`.

### CODE REVIEW

File: packages/internal/dex/sdk-sample-app/src/components/Example.tsx [hunk 1]
```diff
@@ -35,10 +37,10 @@ const allTokens: Token[] = [
   { symbol: 'zkYEET', address: '0x8AC26EfCbf5D700b37A27aA00E6934e6904e7B8e' },
 ];
 
-const buildExchange = (
-  secondaryFeeRecipient: string,
-  secondaryFeePercentage: number
-) => {
+/// @dev Update this to change tokens used in the app
+const allTokens = testnetTokens;
+
```
Comment (Benjimmutable) [inline packages/internal/dex/sdk-sample-app/src/components/Example.tsx:42]: How is this being used?

**Review by Benjimmutable — APPROVED** [review]

File: packages/internal/dex/sdk-sample-app/src/components/Example.tsx (hunk 1 above)
Comment (pano-skylakis) [reply packages/internal/dex/sdk-sample-app/src/components/Example.tsx:42]: It's just to make flipping between the token lists a bit easier. Ideal solution is to have a dropdown that allows selecting the chain, but that's out of scope for this I think

**Comment by pano-skylakis** [comment]
Swap with secondary fees 🎉 https://explorer.immutable.com/tx/0x090cedebf8eea66150d4563e5c1f933d73d1d05a807156472d7c6b7603854eea

**Review by keithbro-imx — APPROVED** [review]

File: packages/internal/dex/sdk/src/constants/tokens/immutable-mainnet.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+export const WIMX_IMMUTABLE_MAINNET: ERC20 = {
+  chainId: IMMUTABLE_MAINNET_CHAIN_ID,
+  address: '0x3a0c2ba54d6cbd3121f01b96dfd20e99d1696c9d',
+  decimals: 18,
+  symbol: 'WIMX',
+  name: 'Wrapped Immutable Token',
+  type: 'erc20',
+};
+
+export const ETH_IMMUTABLE_MAINNET: ERC20 = {
```
Comment (keithbro-imx) [inline packages/internal/dex/sdk/src/constants/tokens/immutable-mainnet.ts:21]: Should this be WETH?

Comment (pano-skylakis) [reply packages/internal/dex/sdk/src/constants/tokens/immutable-mainnet.ts:21]: It's Eth from the contract/block explorer

**Pull request merged**